interface.go

```diff
@@ -87,14 +86,14 @@ type ClientOperator interface {
 	GetAccountInfo(account types.Address, epoch ...*types.Epoch) (accountInfo types.AccountInfo, err error)
 	GetInterestRate(epoch ...*types.Epoch) (intersetRate *hexutil.Big, err error)
 	GetAccumulateInterestRate(epoch ...*types.Epoch) (intersetRate *hexutil.Big, err error)
-	GetBlockRewardInfo(epoch ...*types.Epoch) (rewardInfo []types.RewardInfo, err error)
+	GetBlockRewardInfo(epoch types.Epoch) (rewardInfo []types.RewardInfo, err error)
```

Comment:
why not use point type, since the default epochs are all defined as pointer types.